app/controllers/schools/advice/base_costs_controller.rb

```diff
@@ -72,7 +72,7 @@ def aggregate_meter_mpan_mprn
       end
 
       def aggregate_meter_adapter
-        OpenStruct.new(mpan_mprn: aggregate_meter_mpan_mprn, name_or_mpan_mprn: aggregate_meter_label)
+        OpenStruct.new(mpan_mprn: aggregate_meter_mpan_mprn, name_or_mpan_mprn: aggregate_meter_label, has_readings?: true)
```

Comment:
This is an unrelated change that fixes a bug in the meter breakdown charges on costs page, see similar change for baseload